docs/medical-api/api-reference/document/post-upload-url.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: "Upload Document"
+description: "Returns a presigned URL to use for a medical document upload to our servers."
+api: "POST /medical/v1/document/upload"
+---
+
+This endpoint returns a presigned URL to enable you to upload your patients' medical documents with specified context to our servers, making them available to other HIEs.
+
+## Process
+
+To use this endpoint, you will need to follow these steps:
+
+1. Create a prelimiary version of a Partial\<[DocumentReference](/medical-api/fhir/resources/documentreference)\> to provide context for the document. In this DocumentReference, we made it a mandatory practice to include the `type` field where you can specify the kind of document (using [LOINC](https://www.hl7.org/fhir/valueset-doc-typecodes.html), if possible),`description` field to outline a very brief description of the document, and `context` field to provide the relevant dates. Besides these mandatory fields, we ask that you try to provide as much context to the file as possible, including the practitioner (`author` field in the DocumentReference) and narrative (`text`).
```

Comment:
Things to consider:

- let's not use lang-specific concepts in our technical docs, `Partial<x>` only means something if you're familiar with TS
- why don't we further simplify this, it's still a large paragraph that's hard to grok and contains too much info, that is described in more detail in the sections below ; how about:

```
1. `POST` a `DocumentReference` to describe the document that's being uploaded (see sections below for details on the expected fields and format.
```